.vscode/settings.json

```diff
@@ -25,6 +25,7 @@
         "numrange",
         "postgis",
         "postgres",
+        "postgresql",
```

Comment:
Indentation.

Reply:
What exactly is the problem?

Reply:
Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine.

Reply:
Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine.

Reply:
Looks like the problem is in the mobile app which ignores indentation for all lines around. In the received email notification it's fine. Plus it doesn't allow to reply you here.

Reply:
OK